core/src/main/java/io/questdb/cairo/TableUtils.java

```diff
@@ -100,9 +102,10 @@ public final class TableUtils {
     public static final String META_SWAP_FILE_NAME = "_meta.swp";
     public static final int MIN_INDEX_VALUE_BLOCK_SIZE = Numbers.ceilPow2(4);
     public static final int NULL_LEN = -1;
-    public static final String RESTORE_SNAPSHOT_TRIGGER_FILE_NAME = "_restore";
-    public static final String SNAPSHOT_META_FILE_NAME = "_snapshot";
-    public static final String SNAPSHOT_META_FILE_NAME_TXT = "_snapshot.txt";
+    public static final String RESTORE_FROM_CHECKPOINT_TRIGGER_FILE_NAME = "_restore";
+    public static final String CHECKPOINT_LEGACY_META_FILE_NAME = "_snapshot";
+    public static final String CHECKPOINT_LEGACY_META_FILE_NAME_TXT = "_snapshot.txt";
+    public static final String CHECKPOINT_META_FILE_NAME = "_checkpoint_mata.d";
```

Comment:
Typo here:
```suggestion
    public static final String CHECKPOINT_META_FILE_NAME = "_checkpoint_meta.d";
```

Reply:
thanks Andrei :)

Reply:
fixed in https://github.com/questdb/questdb/pull/4821/commits/54be271306f73d2a04f6caf59638838b561fbaa1